src/main/java/home/DirectionsController.java

```diff
@@ -688,8 +688,8 @@ public void generateQRCode(ArrayList<String> directions) throws WriterException,
     @FXML
     public void showQRCode() {
         if (qrCodeVbox.getMaxHeight() == 0) {
-            qrCodeVbox.setPrefHeight(350);
-            qrCodeVbox.setMaxHeight(350);
+            qrCodeVbox.setPrefHeight(750);
+            qrCodeVbox.setMaxHeight(750);
```

Comment:
750?

Reply:
Yeah, otherwise the really big boi qr codes for cross hospital navigation clip off of the screen